webapp/pages/post/_id/_slug/index.vue

```diff
@@ -90,7 +90,17 @@
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <comment-form
+          v-if="showNewCommentForm && !post.author.blocked"
+          :post="post"
+          @createComment="createComment"
+        />
+        <ds-placeholder v-else>
+          {{ $t('settings.blocked-users.explanation.commenting-disabled') }}
+          <br />
+          {{ $t('settings.blocked-users.explanation.commenting-explanation') }}
+          <a href="https://support.human-connection.org/kb/" target="_blank">FAQ</a>
+        </ds-placeholder>
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-29T16:09:11Z" title="Wednesday, January 29th 2020, 5:09:11 pm +01:00">Jan 29, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -90,7 +90,17 @@
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <comment-form
+          v-if="showNewCommentForm && !post.author.isBlocked"
+          :post="post"
+          @createComment="createComment"
+        />
+        <ds-placeholder v-else>
+          {{ $t('settings.blocked-users.explanation.commenting-disabled') }}
+          <br />
+          {{ $t('settings.blocked-users.explanation.commenting-explanation') }}
+          <a href="https://support.human-connection.org/kb/">FAQ</a>
+        </ds-placeholder>
```


@mattwr18, you've been asking about the placeholder a few times – of course I'm not extremely happy about it since we're trying to get rid of styleguide components when we can. But in this case I think a) it looks fine, b) I don't know a better (and reusable!) solution off the top of my head and c) we'll have to migrate the `ds-placeholder` anyway, so we can leave it for future-us. :grin:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-29T20:07:16Z" title="Wednesday, January 29th 2020, 9:07:16 pm +01:00">Jan 29, 2020</time>_

---


ok, I looked at in the styleguide and it's quite simple, but we can look at it, and design it later